Source/WebKit/UIProcess/API/glib/WebKitDownload.cpp

```diff
@@ -126,9 +135,23 @@ static void webkitDownloadGetProperty(GObject* object, guint propId, GValue* val
     }
 }
 
+static void maybeFinishDecideDestination(WebKitDownload* download)
+{
+    if (download->priv->decideDestinationCallback) {
```

Comment:
if (auto completionHandler = std::exchange(download->priv->decideDestinationCallback, nullptr)) {

Reply:
Anyway, I'll change this as you suggest.